src/main/kotlin/step4/racingcar/racingcar/CarRacing.kt

```diff
@@ -0,0 +1,22 @@
+package step4.racingcar.racingcar
+
+class CarRacing(racers: List<String>, private val moveCount: Int) {
+
+    private val car = racers.map { Car(it) }
+
+    val gameResult: GameResult
+        get() = GameResult(car, moveCount)
+
+    fun execute(randomNumber: NumberGenerator) {
```

Comment:
```suggestion
    fun execute(numberGenerator: NumberGenerator) {
```

CarRacing은 random 값에 대한 정책을 알지 못해도 된다고 생각합니다. randomNumber 보다 더 좋은 이름이 있지 않을까요? 🤔

자동차의 움직임에 맞는 이름이면 더 좋을 것 같아요!